.github/ISSUE_TEMPLATE/question.md

```diff
@@ -11,6 +11,10 @@ assignees: ''
 
 A clear and concise description of what the question is.
 
+**Reproducible Code**
+
+Please submit a reproducible code and tell us how to reproduce it like how he send a request or send what request?
```

Comment:
how you send

Reply:
> how you send

done, thx.

Reply:
Please construct a minimum complete and reproducible example for us to get the same error. And tell us how to reproduce it like how you send a request or send what request

Reply:
done